sigle/src/ui/Button.ts

```diff
@@ -13,12 +13,17 @@ export const Button = styled('button', {
     pointerEvents: 'none',
   },
 
+  '@xl': {
```

Comment:
xl or md?

Reply:
following the breakpoints on typography which are changing at `xl` but maybe it should be `md`? as most other component styles are updating at this breakpoint. Wdyt?

Reply:
I think md make more sense, xl is really for big screens

Reply:
yeah makes sense - should I also update the typography breakpoints here, or do it in a separate pr?

Reply:
here sounds good 👍